woocommerce/woocommerce-android
# [Blaze] Product details CTA

Closes: #9977 

### Description
This PR adds the Blaze CTA to the Product Details screen.

**Note:** The commit cec213d14211ec4bff4dda4a7bbd3d3f04c8b301 is just a small optimization, it removes a duplicate image, so it's better to review it individually.

### Testing instructions
1. Using a Jetpack site.
2. Open product details.
3. Confirm the Blaze CTA card is shown.

### Images/gif
<img width=320 src="https://github.com/woocommerce/woocommerce-android/assets/1657201/8b4bc422-c0a6-4cad-a2b8-e1e1bd5cea44"/>
